### ISSUE TITLE
Wrong rating?

### ISSUE THREAD

**Issue by CasperNielsen** [mynttt/UpdateTool#19]
Having some problems with some shows, the ratings is way to low.

Eks. (There is more shows than this)

[INFO ] - 2020-03-18 13:57:30 @ ImdbTransformer.updateMetadata: Adjust rating: 7.5 -> 3.4 for Star Trek: Picard
[INFO ] - 2020-03-18 13:57:30 @ ImdbTransformer.updateMetadata: Adjust rating: 8.5 -> 5.0 for Prodigal Son

Where is the tool getting these low ratings from? :)

**Comment by mynttt** [comment]
Do you use DVD order with your plex library?

I can look into it but you'll have to upload the following files to pastebin:

- cache-tvdb2imdb.json
- updatetool.log with the highest number in your folder

Remove the API details within the first few lines of the log if you don't want to expose your credentials.

**Comment by mynttt** [comment]
Also helpful would be the output of running this query on your plex database:

```sql
SELECT * FROM metadata_items WHERE title = 'Star Trek: Picard';
```

**Comment by CasperNielsen** [comment]
[Compressed file.zip](https://github.com/mynttt/UpdateTool/files/4350895/Compressed.file.zip)

Here is the log files and SQL output :)

Not sure about the DVD order, have never heard of that in Plex.

**Comment by mynttt** [comment]
I guess DVD order will not be a problem then, Plex defaults to Aired which this tool uses to resolve TVDB id's.

My guess is that for some reason there is some mismatch or wrong involvement happening, I'll investigate it tomorrow and find the cause hopefully.

**Comment by mynttt** [comment]
Issue seems to that the TV series are TMDB matched which I was not aware of (my TV series are all TVDB matched).

So these TV series are probably treated as movies which would explain the wrong ratings.

I would investigate further tomorrow, it would be helpful to get your TV show database as SQL dump, so I can insert it into my development copy to reproduce your library behavior.

Another csv dump with this query would help me with that.

'''sql
SELECT * FROM metadata_items WHERE library_section_id = 1;
'''

Thanks for bringing this issue up! This definitely needs to be fixed.

**Comment by mynttt** [comment]
Has been fixed with f1e2bdd972e75958c6e76cd996f9341b3d71ec04.

The issue was that your Library reported being of TVDB but had some items TMDB matched. So the pipeline tried to resolve these TMDB items via TVDB which does not work at all.

TV Show matching for TMDB has been added now and the tool will now also process libraries that use the TMDB agent for TV shows.

One run should fix your mismatched ratings again.
